Appraisals

```diff
@@ -9,18 +9,41 @@ appraise 'rails-6.0' do
   gem 'railties', '~> 6.0.0'
   gem 'minitest-rails', '~> 6.0.0'
 
-  # net-smtp, net-imap and net-pop were removed from default gems in Ruby 3.1, but is used by the `mail` gem.
-  # So we need to add them as dependencies until `mail` is fixed: https://github.com/mikel/mail/pull/1439
-  # Taken from https://github.com/rails/rails/pull/42366
-  gem "net-smtp", require: false
+  # net-smtp, net-imap and net-pop were removed from default gems in Ruby 3.1,
+  # but is used by the `mail` gem. So we need to add them as dependencies until
+  # `mail` is fixed: https://github.com/mikel/mail/pull/1439 Taken from
+  # https://github.com/rails/rails/pull/42366
+  gem 'net-smtp', require: false
 end
 
 appraise 'rails-6.1' do
   gem 'railties', '~> 6.1.0'
   gem 'minitest-rails', '~> 6.1.0'
 
-  # net-smtp, net-imap and net-pop were removed from default gems in Ruby 3.1, but is used by the `mail` gem.
-  # So we need to add them as dependencies until `mail` is fixed: https://github.com/mikel/mail/pull/1439
-  # Taken from https://github.com/rails/rails/pull/42366
-  gem "net-smtp", require: false
+  # net-smtp, net-imap and net-pop were removed from default gems in Ruby 3.1,
+  # but is used by the `mail` gem. So we need to add them as dependencies until
+  # `mail` is fixed: https://github.com/mikel/mail/pull/1439 Taken from
+  # https://github.com/rails/rails/pull/42366
+  gem 'net-smtp', require: false
+end
+
+appraise 'rails-7.0' do
+  gem 'railties', '~> 7.0.0'
+  gem 'devise', '>= 4.8.1' # devise introduced Rails 7 support
+  gem 'minitest-rails', github: 'blowmage/minitest-rails', branch: 'master' # temporary until official release of Rails 7 support for minitest-rails
```

Comment:
The official release already available:

https://github.com/minitest/minitest-rails/releases/tag/v7.0.0